plugins/elasticsearch_logs.yaml

```diff
@@ -0,0 +1,98 @@
+version: 0.0.1
+title: Elasticsearch
+description: Log parser for Elasticsearch
+parameters:
+  - name: enable_json_logs
+    type: bool
+    default: true
+  - name: json_log_paths
+    type: "[]string"
+    default:
+      - "/var/log/elasticsearch/*_server.json"
+      - "/var/log/elasticsearch/*_deprecation.json"
+      - "/var/log/elasticsearch/*_index_search_slowlog.json"
+      - "/var/log/elasticsearch/*_index_indexing_slowlog.json"
+      - "/var/log/elasticsearch/*_audit.json"
+  - name: enable_gc_logs
+    type: bool
+    default: false
+  - name: gc_log_paths
+    type: "[]string"
+    default:
+      - "/var/log/elasticsearch/gc.log*"
+  - name: start_at
+    type: string
+    supported:
+      - beginning
+      - end
+    default: end
+template: |
+  receivers:
+    {{if .enable_json_logs}}
```

Comment:
Hmm I believe that we wanted to move away from flags that required a boolean switch to decide whether to parse as JSON or regex. I was able to do it here for mongodb https://github.com/observIQ/stanza-plugins/blob/40f49c1f9a87c4ba71b4522c30a55ce89d4402d2/plugins/mongodb.yaml#L91-L96

Is this something we want to do going forward? 


https://github.com/observIQ/stanza-plugins/pull/386
https://github.com/observIQ/stanza-plugins/pull/388

Reply:
First push is to just get these ported then we can iterate on standards.